regular_expressions_boundaries_anchors/regular_expressions_boundaries_anchors.md

```diff
@@ -0,0 +1,425 @@
+<!--
+
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 1.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+
+title: Regular Expressions: Flags, Anchors, and Boundaries
+
+comment:  Use flags, anchors, and boundaries in regular expressions, or regex, for complex pattern matching.
+
+long_description: Learn to compose intermediate regular expressions using flags, anchors, boundaries, and more, in order to find text that matches patterns you describe.
+
+estimated_time_in_minutes: 45
+
+@pre_reqs
+Learners should have some experience composing and using moderately complex regular expressions (regex).  For an introduction to regular expression concepts, consider the [Demystifying Regular Expressions](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_regular_expressions/demystifying_regular_expressions.md#1) module.  To learn how to compose and use simple regular expressions, consider the [Regular Expressions Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/regular_expressions_basics/regular_expressions_basics.md#1) module.  We refer to groups in this module, which are covered in the [Regular Expressions: Groups](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/regular_expressions_groups/regular_expressions_groups.md#1) module. 
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain what a regular expression flag does
+- Use anchors and boundaries in regular expressions
+- Use boundaries in regular expressions
+
+@end
+good_first_module: false
+coding_required: true
+coding_level: intermediate
+sequence_name: regex
+previous_sequential_module: regular_expressions_groups
+@sets_you_up_for
+- regular_expressions_lookaheads
```

Comment:
```suggestion

```
I don't think these modules actually depend on/set you up for each other, even though they are sequential in the sequence, which is recorded elsewhere in the metadata.

Reply:
agree!  adding to a batch for commit